<?xml version="1.0"?>
<launch>
    <node ns="iiwa" 
          pkg="pkg_ijcai2019" 
          type="demopath_manual" 
          name="demopath_manual" 
          output="screen" respawn="false"/>

    <rosparam param="changkyu/pkg_ijcai2019/demopath_manual" command="load" file="$(find pkg_ijcai2019)/config/input.yaml" />
</launch>
